src/vm.rs

```diff
@@ -642,7 +658,18 @@ impl<T: GuestStagePageTable> Vm<T, VmStateFinalized> {
                                     stop_flags,
                                 });
                                 // Safety: PmuFunction does not touch memory.
-                                unsafe { ecall_send(&msg) }
+                                match unsafe { ecall_send(&msg) } {
+                                    Ok(_) => {if counter_index_base < 32 {
```

Comment:
This formatting seems strange, but the fmt check is passing. What's going on in this file that is causing that?

Reply:
Looks like rustfmt is funky when it comes to match arms or something like that. I removed some conditionals, and it does look better now.